src/utils/components/EnvironmentEditor/tests/mosks.ts

```diff
@@ -0,0 +1,114 @@
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+
+export const exampleVirtualMachineWithEnvironments: V1VirtualMachine = {
+  apiVersion: 'kubevirt.io/v1',
+  kind: 'VirtualMachine',
+  metadata: {
+    annotations: {
+      'vm.kubevirt.io/validations':
+        '[\n  {\n    "name": "minimal-required-memory",\n    "path": "jsonpath::.spec.domain.resources.requests.memory",\n    "rule": "integer",\n    "message": "This VM requires more memory.",\n    "min": 1610612736\n  }\n]\n',
```

Comment:
Literal strong won't work here instead of new line escape?

Reply:
It works! pushing...